docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md

```diff
@@ -402,7 +492,8 @@ Follow the steps below to install the Merchant Portal Core feature frontend.
 ### Prerequisites
 
 Environment requirements:
-  * NodeJs v12+
+  * NPM v6 (higher versions have problems with workspace)
+  * NodeJs v12-14
```

Comment:
@danilmoro3 Please verify for correctness.

Reply:
Looks correct